profiles/nuvla.json

```diff
@@ -0,0 +1,129 @@
+{
+  "name": "Nuvla",
+  "revision": "2015-10-13",
+  "vendor_verified": "2015-10-13",
+  "url": "http://nuv.la/login",
+  "status": "production",
+  "status_since": "2012-01-01",
+  "type": "Generic",
```

Comment:
If you take a look at the link I provided you'll see that I don't refer to IaaS when talking about IaaS-centric. It's still all PaaS that is listed here.